src/code/UNOClasses/Deck.java

```diff
@@ -55,4 +56,34 @@ public void shuffleDeck() {
     public int deckTotal() {
         return deck.size();
     }
-}
+
+    public UNOCard deal() {
+        return deck.remove(0);
```

Comment:
This would be essentially taking the bottom card on the deck and giving it to a player, whether it be dealing or the player drawing. I don't think this will break the game if we keep it like this but maybe we want to remove the top card on the deck? (Last card in the deck vector)

Reply:
Ha, I guess I was thinking of it as the top card.

Reply:
Since we are using vector and not stack I am not sure which should be considered top and which would be considered the bottom.

Reply:
Yeah, and like I said, it's not going to make or break the game and I'm fine if we leave it this way